### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

File: src/client/all_meals.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const [isEmpty, setEmpty] = useState(true);
+
+  const apiBaseUrl = "http://localhost:3000/api/meals";
+
+  const SearchUser = useCallback((value) => {
+    setUsers([]);
+    setEmpty(true);
+
+    console.log("in search user");
+    let apiUrl = "http://localhost:3000/api/meals";
```
Comment (islam-fawzy25) [inline src/client/all_meals.js]: we have to fetch from /api/meals since we deployed so we will not use the local data

**Review by islam-fawzy25 — APPROVED** [review]
Hi Parul,
Nicly done 👍🏻 
just you need to modify the URL as I mentioned afterwards you will find the data 
you are welcome if you have any questions

File: src/client/all_meals.js (hunk 1 above)
Comment (hparul) [reply src/client/all_meals.js]: done!thanks for the feedback!

**Review by dsamourkasidis — APPROVED** [review]
Well done Parul :+1: 
The main features of the app are there, and some extra as well, good job you're good to go!
Good overall work with the components, the props and using hooks.

As for future improvements, I would suggest you create two more components to make some features more 'separate'.
- I would create a new component for adding a reservation which would include the form for adding a new reservation instead of having the form inside the MealsWithId component.
- I would do the same thing for the reviews, a new component that lists the reviews and has a form to add a new one.

That way the MealsWithId component becomes more clear to read and it's easier to locate everything.

**Pull request merged**